### ISSUE TITLE
Can't recovery pharse/private key

### ISSUE THREAD

**Issue by pisceans** [Meteor-Wallet/meteor-public#9]
Private key/ pharse not connected on account and can't add new key. How to fixed it?
Please answer me asap !

**Comment by Festivo123** [comment]
I think i have the same error, were you able to fix it already?

**Comment by pisceans** [comment]
No, I've been waiting all day to add a new access key but there's still no change,
There is only a loading image

**Comment by Festivo123** [comment]
Yes, same for me. Did you try the support chat yet?

**Comment by pisceans** [comment]
Yes I have contacted them and you know what they say "try again and wait" 
And what I'm thinking is this is crazy I tried waiting all day and repeating it for a few days and they just say try again

**Comment by Festivo123** [comment]
I have exactly the same problem. I have told them the error is inside the wallet. I think the wallet was made in a time of server error or something, so the private key was incorrectly linked to the wallet. 
I really hope they will respond to me with an appropriate solution. When I hear something I will let you know!

**Comment by pisceans** [comment]
Thank you for your attention, I appreciate it
